docs/lro-status-codes-return-type-schema.md

```diff
@@ -2,7 +2,7 @@
 
 ## Category
 
-SDK Warning
+SDK Error
```

Comment:
Question on this one. We now have support in autorest, and more recently in aaz-dev-tools (CLI generator), for the `final-state-schema` property in `x-ms-long-running-operation-options`:

https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-long-running-operation

If a service uses this new option, it does not need to "lie" about returning a 200 or 201 response from a long-running operation. So do we really want to make this case an error?

Reply:
The rule is scoped to Put operations only. And it states that if a 200 or a 201 are specified then they need a schema to be defined

Reply:
Okay. It would be nice if the rule description actually said this. But I suppose that can be fixed later.